### ISSUE TITLE
[Quest][Northrend] A Tangled Skein - Improvement which could be made.

### ISSUE THREAD

**Issue by Aridam** [dalaranwow/dalaran-wow#5152]
**Current behaviour**: Upon channeling the Quest Item's Cast on the Plague Sprayers all the way through, they instantly break-apart mid-air and stay so as they despawn.

**Expected behaviour**: https://youtu.be/B1byxXXQTnQ?t=29 and onward.

**Steps to reproduce the problem**:

As seen in the video above.

**Include proofs for this behaviour**

Quest's ID: https://wowgaming.altervista.org/aowow/?quest=12555
Quest Item's ID: https://wowgaming.altervista.org/aowow/?item=38515
Plague Sprayer's ID: https://wowgaming.altervista.org/aowow/?npc=28274

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Aridam,
Issue confirmed on PTR.
It's mostly an animation issue, but can be fixed. After some messing around with the script of that creature, I've found out that reason for the broken plague sprayer(it's GO) to spawn in the air is that the creature dies and there is no time for it to fall. But SAI action lists doesn't work for already dead creatures so my suggestion is to change the inhabit_type within the script so it can fall (in teory) and then kill it on the ground.
I've tried with this:
`` sql
-- Actionlist SAI
SET @ENTRY := 2827400;
DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=9;
INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
(@ENTRY,9,0,0,0,0,100,0,500,500,0,0,11,53236,0,0,0,0,0,1,0,0,0,0,0,0,0,"Plague Sprayer - On Script - Cast 'Plague Sprayer: Huge Explosion'"),
(@ENTRY,9,1,0,0,0,100,0,500,500,0,0,60,0,0,0,0,0,0,1,0,0,0,0,0,0,0,"Plague Sprayer - On Script - Set Fly Off"),
(@ENTRY,9,2,0,0,0,100,0,300,300,0,0,33,28289,0,0,0,0,0,12,1,0,0,0,0,0,0,"Plague Sprayer - On Script - Quest Credit ''"),
(@ENTRY,9,3,0,0,0,100,0,5000,5000,0,0,11,51314,0,0,0,0,0,1,0,0,0,0,0,0,0,"Plague Sprayer - On Script - Cast 'A Tangled Skein: Summon Broken Plague Sprayer'"),
(@ENTRY,9,4,0,0,0,100,0,0,0,0,0,37,0,0,0,0,0,0,1,0,0,0,0,0,0,0,"Plague Sprayer - On Script - Kill Self"),
(@ENTRY,9,5,0,0,0,100,0,0,0,0,0,47,0,0,0,0,0,0,1,0,0,0,0,0,0,0,"Plague Sprayer - On Script - Set Visibility Off");
```

But the fly off doesn't work for some reason.
